src/alpaca_eval/models_configs/cohere-chat/configs.yaml

```diff
@@ -1,8 +0,0 @@
-cohere-chat:
-  prompt_template: "cohere/prompt.txt"
-  fn_completions: "cohere_completions"
-  completions_kwargs:
-    model_name: "command"
-    mode: "chat"
-    max_tokens: 2048
-  pretty_name: "Cohere Chat"
```

Comment:
Why remove the chat model?
if it's because it was updated, can we get the updated results on the leaderboard instead?

Reply:
The chat mode of the command model was not really a different model before, just a different prompting style. These have been further unified (essentially turning plain instructions into single-turn chats), making them not worth listing as separate entries.
Having a single entry also will make it easier to give you more regular updates.

Reply:
I see, so `client.chat` is being depreciated?
Then let's remove that from the code also?

Reply:
It's more for multi-turn chats, indeed less appropriate for tests like this. Removed in [eb52df3](https://github.com/tatsu-lab/alpaca_eval/pull/141/commits/eb52df3f5a8a3eec420792f6c2587bd397858b19)